### ISSUE TITLE
Switch from pyGame to pySDL2.

### ISSUE THREAD

**Issue by mdsitton** [mdsitton/fofix#5]
This switch will be required in order to use newer opengl versions for rendering.PyGame is also pretty out of date and is still based on the original SDL. I had already done a bit of work to get away from certain pyGame features back in the main FoFiX repo. As far as i know the main things i will need to figure out is the font rendering system, and the input system.

**Comment by poVoq** [comment]
Great would be if it would run directly on systems only capable of Open GL ES, like those many ARM boards (Rasberry PI2, ODROID etc.).

**Comment by mdsitton** [comment]
Yeah, that would be nice. But a quite a bit of work would be required to get to that point, not impossible though.
